dmscripts/bulk_upload_documents.py

```diff
@@ -0,0 +1,39 @@
+import os
+import re
+import sys
+sys.path.insert(0, '.')
+from dmutils.documents import get_countersigned_agreement_document_path
+
+
+def upload_file(bucket, dry_run, file_path, framework_slug):
+    supplier_id = get_supplier_id_from_framework_file_path(file_path)
+    upload_path = get_countersigned_agreement_document_path(framework_slug, supplier_id)
+    if not dry_run:
+        with open(file_path) as file_contents:
+            bucket.save(upload_path, file_contents, acl='public-read', move_prefix=None)
+        print(supplier_id)
+    else:
+        print("[Dry-run] UPLOAD: '{}' to '{}'".format(file_path, upload_path))
+
+
+def get_bucket_name(stage, document_type):
+    bucket_name = 'digitalmarketplace-{0}-{1}-{1}'.format(document_type, stage)
+    print("BUCKET: {}".format(bucket_name))
+    return bucket_name
+
+
+def get_all_files_of_type(local_directory, file_type):
+    for root, subfolder, files in os.walk(local_directory):
+        for filename in files:
+            if filename.endswith(file_type):
+                yield os.path.join(root, filename)
+
+
+def get_supplier_id_from_framework_file_path(path):
+    # Filenames from CCS are in the format:
```

Comment:
Now this is more generic this comment is misleading.  With the new regex the only thing we can say about the file path is something like
```
# All files to be uploaded must have a dash followed by a supplier id
# The rest of the filename is disregarded 
```
This worries me slightly... it means there is a need to be extra-careful that only the types of file we want to upload are in the folder. 

I don't know if it would be possible to have another parameter that defines a bit more narrowly which files should be uploaded... it's probably too tricky to do in a robust way, but we'll just need to be careful about the folders we run this against.